src/pages/_app.page.tsx

```diff
@@ -37,17 +41,20 @@ function MyApp({ Component, pageProps }: AppProps): JSX.Element {
          <Head>
             <title>Caribbean Online Judge</title>
          </Head>
+         <CssBaseline />
          <Provider store={store}>
             <ThemeProvider theme={theme}>
-               {router.pathname.startsWith("/account") ? (
```

Comment:
Please move this logic to an independent function to get more clear render function